tests/lib/outputs/ocsf/ocsf_test.py

```diff
@@ -87,6 +99,36 @@ def test_transform(self):
             "compliance": findings[0].compliance,
         }
 
+    def test_validate_ocsf(self):
+        mock_file = StringIO()
+        findings = [
+            generate_finding_output(
+                status="FAIL",
+                severity="low",
+                muted=False,
+                region=AWS_REGION_EU_WEST_1,
+                timestamp=datetime.now(),
+                resource_details="resource_details",
+                resource_name="resource_name",
+                resource_uid="resource-id",
+                status_extended="status extended",
+            )
+        ]
+
+        output = OCSF(findings)
+        output._file_descriptor = mock_file
+
+        with patch.object(mock_file, "close", return_value=None):
+            output.batch_write_data_to_file()
+
+        mock_file.seek(0)
+        content = mock_file.read()
```

Comment:
do you need to write the contents to that mock object? You can just load the object into the request as a JSON body.